lightningd/gossip_control.c

```diff
@@ -221,8 +221,73 @@ static const struct json_command getnodes_command = {
     "Returns a list of all nodes that we know about"};
 AUTODATA(json_command, &getnodes_command);
 
+static bool json_getroute_reply(struct subd *gossip, const u8 *reply, const int *fds,
+				struct command *cmd)
+{
+	struct json_result *response;
+	struct route_hop *hops;
+	size_t i;
+
+	fromwire_gossip_getroute_reply(reply, reply, NULL, &hops);
```

Comment:
It's good practice to check fromwire_ results; if we get somthing weird from a daemon we should at least log it...